posts/Andrea/2019-07-29-hibernate-orm-544-final-release.adoc

```diff
@@ -0,0 +1,32 @@
+= "Hibernate ORM 5.4.4.Final released"
+Andrea Boriero
+:awestruct-tags: [ "Hibernate ORM", "Releases" ]
+:awestruct-layout: blog-post
+:released-version: 5.4.4.Final
+:release-id: 31774
+---
+
+We just released the fourth maintenance release of Hibernate ORM 5.4.
+
+It is designed to be a drop in replacement for ORM 5.4.3.
+
+== What's new
+
+This release other than fixing more bugs introduces a new feature that allow enhanced entities to be returned in a completely uninitialized state.  See http://in.relation.to/2019/07/29/bytecode-proxy[Bytecode enhancement as proxy] for details on this new feature.
```

Comment:
> This release other than fixing more bugs 

I wouldn't say this, it just makes this sentence too long. Also it's not accurate as we didn't "just" fix more bugs, we did a ton of things ;)

Reply:
A quick alternative (just an idea, not a strong preference):

> This release allows returning uninitialised enhanced entities.

It also removes the passive form that I don't like that much in this case :)
You could also add a one liner on why it's useful.

My 0.02 cents.

Reply:
+1

please describe why it's useful, and how to use it.. can keep it short if you / Steve are planning a more extensive blog on the subject?

But at least describe it briefly

Reply:
there is the link to an the blog post where the Steve describes the feature

Reply:
the PR https://github.com/hibernate/in.relation.to/pull/134 I mentioned in description